web/index.php

```diff
@@ -11,14 +11,21 @@
  * @see https://github.com/opencfp/opencfp
  */
 
-require_once __DIR__ . '/../vendor/autoload.php';
+namespace OpenCFP;
+
+use Symfony\Component\Debug\Debug;
+use Symfony\Component\HttpFoundation\Request;
 
-use OpenCFP\Application;
-use OpenCFP\Environment;
+require_once __DIR__ . '/../vendor/autoload.php';
 
-$basePath    = \realpath(\dirname(__DIR__));
 $environment = Environment::fromServer($_SERVER);
 
-$app = new Application($basePath, $environment);
+if (!$environment->isProduction()) {
+    Debug::enable();
```

Comment:
Isn't this doing the same as my other comment?

Reply:
This just registers a nice error handler that kicks in when the kernel cannot be booted. So for instance, if I break the DI configuration, I get this

<img width="931" alt="bildschirmfoto 2017-12-16 um 14 57 27" src="https://user-images.githubusercontent.com/1506493/34071114-b65da568-e271-11e7-8ac0-61412b3a8a65.png">

instead of this.

<img width="930" alt="bildschirmfoto 2017-12-16 um 14 57 04" src="https://user-images.githubusercontent.com/1506493/34071117-bde43fe0-e271-11e7-9df7-3b4dc2cc9bec.png">

Reply:
Moved to #963.